### PR TITLE
Added channels, changed orders to keep the same hierarchy the server has, modified the readme's and obsolete channel descriptions, added graven to reférents lists

### PR DESCRIPTION
**Pull request by padmeon**
This commit is nearly a complete overhaul of this repository.
- Added channels : roles, questions-fréquentes, anciennes-vidéos, clash of code, dart | flutter

- Changed the orders of the different channels to keep the same hierarchy since new channels were added, example: 
the channel `annonces` was number two and became number 4 because of the channels `roles` and `questions-fréquentes`

- Added the new channels to the readme's

- Modified some obsolete descriptions of channels (like the `autres` channel that doesn't exist anymore because it is now a forum) Deleted some channels that don't exist/are not visible anymore

- Added Graven in the référents lists (like `applications-mobiles` and `dart | flutter`)

Good luck reviewing everything 😉

### CODE REVIEW

File: channels/1 - Système/2_roles.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# `#🎭︱roles`
+Ce salon contient un système de
```
Comment (g-clems) [inline channels/1 - Système/2_roles.md]: Description incomplète

File: channels/4 - Salons d'aide/16_autres.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# `#❔︱autres`
-Le salon `#❔︱autres` est un salon vous permettant de poser vos questions sur les 
-langages non-couverts par un salon
+# `#❔︱autre︱other`
+Le salon `#❔︱autre︱other` est un forum vous permettant de poser vos questions, sur les
```
Comment (g-clems) [inline channels/4 - Salons d'aide/16_autres.md]: virgule en trop

File: channels/4 - Salons d'aide/16_autres.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# `#❔︱autres`
-Le salon `#❔︱autres` est un salon vous permettant de poser vos questions sur les 
-langages non-couverts par un salon
+# `#❔︱autre︱other`
+Le salon `#❔︱autre︱other` est un forum vous permettant de poser vos questions, sur les 
+langages non-couverts par un salon, sous la forme de posts
```
Comment (g-clems) [inline channels/4 - Salons d'aide/16_autres.md:4]: Guidelines manquante

**Review by g-clems — CHANGES_REQUESTED** [review]

File: channels/4 - Salons d'aide/16_autres.md (hunk 3 above)
Comment (padmeon) [reply channels/4 - Salons d'aide/16_autres.md:4]: Qu'est ce qu'une guideline?

Comment (padmeon) [reply channels/4 - Salons d'aide/16_autres.md:4]: C'est d'expliquer comment on ouvre un post?

File: channels/1 - Système/2_roles.md (hunk 1 above)
Comment (SkyrixFr) [reply channels/1 - Système/2_roles.md:2]:
```suggestion
Ce salon contient un système de rôles permettant aux utilisateurs de définir les languages pratiqué par les membres du serveur
```

File: channels/4 - Salons d'aide/16_autres.md (hunk 3 above)
Comment (g-clems) [reply channels/4 - Salons d'aide/16_autres.md:4]: Elles sont visible en cliquant sur "nouveau post"

**Comment by padmeon** [comment]
All the changes asked were made and more was done (things I forgot to add in the old commit)

**Review by g-clems — APPROVED** [review]

**Review by RedsTom — APPROVED** [review]

**Pull request merged**